src/components/small_bottom_bar/ribbon_menu/ribbon_menu.ts

```diff
@@ -51,6 +52,7 @@ export class RibbonMenu extends Component<RibbonMenuProps, SpreadsheetChildEnv>
       this.state.parentState = { ...this.state };
       this.state.menuItems = children;
       this.state.title = menu.name(this.env);
+      this.containerRef.el?.scrollTo({ top: 0 });
```

Comment:
If we want to be really fancy, we could store a scroll for each menu, so we keep the scroll when going back to the parent menu. Probably not worth tho